doc/changes/10710.md

```diff
@@ -0,0 +1 @@
+- Enable dune's cache by default. (#10710, @ElectreAAS, joint work with @Leonidas-from-XIV)
```

Comment:
Probably worth mentioning the new `DUNE_CACHE_RULES` environment variable.

Reply:
Fixed now :)